build.gradle

```diff
@@ -170,37 +170,39 @@ dependencies {
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-webflux'
-  implementation group: 'io.springfox', name: 'springfox-boot-starter', version: '3.0.0'
-
-  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: reformLoggingVersion
-  implementation group: 'com.github.hmcts.java-logging', name: 'logging-appinsights', version: reformLoggingVersion
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-data-jpa'
-  implementation group: 'com.azure.spring', name: 'spring-cloud-azure-starter-active-directory', version: '4.0.0'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-resource-server'
   implementation group: 'org.springframework.boot', name: 'spring-boot-starter-oauth2-client'
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation', version: '2.6.0'
+  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-validation'
+
+  implementation group: 'org.springdoc', name: 'springdoc-openapi-ui', version: '1.6.11'
+
+  implementation group: 'com.github.hmcts.java-logging', name: 'logging', version: reformLoggingVersion
+  implementation group: 'com.github.hmcts.java-logging', name: 'logging-appinsights', version: reformLoggingVersion
+
+  implementation group: 'com.azure.spring', name: 'spring-cloud-azure-starter-active-directory', version: '4.4.0'
   implementation group: 'org.postgresql', name: 'postgresql', version: '42.4.1'
   implementation group: 'com.github.hmcts', name: 'pip-data-models', version: '1.1.1', {
     exclude group: 'org.springframework.boot', module: 'spring-boot-starter-data-jpa'
   }
-  implementation group: 'org.springframework.boot', name: 'spring-boot-starter-webflux', version: '2.6.4'
   implementation group: 'com.squareup.okhttp3', name: 'okhttp', version: '4.9.3'
   implementation group: 'com.vladmihalcea', name: 'hibernate-types-52', version: '2.14.0'
-  implementation group: 'org.apache.commons', name: 'commons-lang3', version: '3.0'
 
-  // Force upgrade snakeyaml version for CVE-2022-25857
+  // Force upgrade snakeyaml version for CVE-2022-38752
   implementation( group: 'org.yaml', name: 'snakeyaml').version {
-    strictly("1.31")
+    strictly("1.33")
   }
 
-  testImplementation(platform('org.junit:junit-bom:5.8.1'))
+  testImplementation(platform('org.junit:junit-bom:5.9.1'))
   testImplementation group: 'org.springframework.boot', name: 'spring-boot-starter-test', {
     exclude group: 'junit', module: 'junit'
     exclude group: 'org.junit.vintage', module: 'junit-vintage-engine'
   }
   implementation 'org.apache.tomcat.embed:tomcat-embed-core:9.0.63'
   implementation 'org.apache.tomcat.embed:tomcat-embed-websocket:9.0.63'
 
+  testImplementation group: 'org.apiguardian', name: 'apiguardian-api', version: '1.1.2'
+
   testImplementation group: 'com.h2database', name: 'h2', version: '2.1.210'
```

Comment:
Just spotted we're still using h2 on subs man. Need to make a ticket to move this over to zonky embeddeddatabase like the others. Futureproofs us on this.

Reply:
Just raised a ticket for this :) https://tools.hmcts.net/jira/browse/PUB-1585